src/Cookie/SetCookie.php

```diff
@@ -69,12 +69,7 @@ public static function fromString(string $cookie): self
      */
     public function __construct(array $data = [])
     {
-        $replaced = \array_replace(self::$defaults, $data);
-        if ($replaced === null) {
```

Comment:
Why did you remove this check?

Reply:
If you revert this change, what will happen then?

Reply:
Phpstan will complain :/

Reply:
You can check master failing build.

Reply:
PHPStan probably things `array_replace` returns an array, not null. Removing the check is not a good idea IMHO, because the function *can* return null, in which case we should throw an exception.

Reply:
OK.. I will leave the exception there.. My major problem here is that this error was reported two days ago.. and two days ago we merged this throw exception. What happened now with PHPStan and it false reports that array_replace returns only `array`?

Reply:
> Removing the check is not a good idea IMHO

reverted..